python/gtsam/tests/test_SfmData.py

```diff
@@ -0,0 +1,81 @@
+"""
+GTSAM Copyright 2010-2019, Georgia Tech Research Corporation,
+Atlanta, Georgia 30332-0415
+All Rights Reserved
+
+See LICENSE for the license information
+
+Unit tests for testing dataset access.
+Author: Frank Dellaert (Python: Sushmita Warrier)
+"""
+# pylint: disable=invalid-name, no-name-in-module, no-member
+
+from __future__ import print_function
+
+import unittest
+
+import numpy as np
+
+import gtsam
+#from gtsam import SfmCamera
+from gtsam.utils.test_case import GtsamTestCase
```

Comment:
I am not sure if this is standard practice for importing in Python. Hasn't gtsam already been imported above?

Reply:
Yeah, other tests have done it too (refer https://github.com/borglab/gtsam/blob/develop/python/gtsam/tests/test_Point2.py)